v0/destinations/candu/transform.js

```diff
@@ -0,0 +1,97 @@
+const { EventType } = require("../../../constants");
+const {
+  CustomError,
+  defaultRequestConfig,
+  constructPayload,
+  getSuccessRespEvents,
+  getErrorRespEvents,
+  removeUndefinedAndNullAndEmptyValues
+} = require("../../util");
+const { endpoint, identifyDataMapping, trackDataMapping } = require("./config");
+
+const responseBuilder = (body, { Config }) => {
+  const payload = body;
```

Comment:
We are making calls even if no relevant fields are present. It would be better to check if there are some minimum fields present in the payload apart from type and context . If no such fields are present we shouldn't make calls.

Reply:
@abandonned Looks like this is taken care of as part of the mapping JSON files (I see some fields are marked `required`).
However, @ItsSudip it's worth double-checking if we have missed anything.

Reply:
oh  I checked it using test cases and no error popped. will check with @ItsSudip

Reply:
Good catch. We should address if that's the case.

Reply:
For identify `traits` and for track `event` along with `type` are required fields, without them, it is throwing error.